.gitignore

```diff
@@ -23,5 +23,11 @@ settings.json
 
 *-debug.log*
 yarn-error.log*
+debug.log
 
 .idea
+
+# firestore emulator
+
+# vscode local history plugin
+.history
```

Comment:
Please kindly revert changes here. Also, a word in general: please [use a global `.gitignore`](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for your plugins. I apologize for the inconvenience, but including each and every IDE plugin in a repo won't scale well.

Reply:
Thank you for the amazing suggestion, did not know about that